react/store/createReturnRequest/CreateReturnRequest.tsx

```diff
@@ -147,10 +147,10 @@ export const CreateReturnRequest = (props: RouteProps) => {
 
 export const CreateReturnRequestContainer = (props: RouteProps) => {
   return (
-    <OrderToReturnProvider>
-      <StoreSettingsPovider>
+    <StoreSettingsPovider>
+      <OrderToReturnProvider>
         <CreateReturnRequest {...props} />
-      </StoreSettingsPovider>
-    </OrderToReturnProvider>
+      </OrderToReturnProvider>
+    </StoreSettingsPovider>
   )
```

Comment:
Due to `validateNewReturnRequestFields` now requiring the new setting, I had to swap providers